tests/e2e/tests/regression/browser/filtering.e2e.ts

```diff
@@ -125,3 +126,69 @@ test
+    .meta({ rte: rte.standalone })
+    ('Verify that when user clicks on “clear” control with no filter per key name applied all characters and filter per key type are removed, “clear” control is disappeared', async t => {
+        keyName = `KeyForSearch${chance.word({ length: 10 })}`;
+        //Set filter by key type and type characters
+        await t.typeText(browserPage.filterByPatterSearchInput, keyName);
+        await browserPage.selectFilterGroupType(COMMAND_GROUP_SET);
+        //Verify the clear control
+        await t.click(browserPage.clearFilterButton);
+        await t.expect(browserPage.multiSearchArea.find(browserPage.cssFilteringLabel).visible).notOk('The filter per key type is removed');
+        await t.expect(browserPage.filterByPatterSearchInput.getAttribute('value')).eql('', 'All characters from filter input are removed');
+        await t.expect(browserPage.clearFilterButton.visible).notOk('The clear control is disappeared');
+    });
+test
+    .after(async() => {
+        //Delete database
+        await deleteDatabase(ossStandaloneConfig.databaseName);
+    })
+    .meta({ rte: rte.standalone })
+    ('Verify that when user clicks on “clear” control and filter per key name is applied all characters and filter per key type are removed, “clear” control is disappeared', async t => {
+        keyName = `KeyForSearch${chance.word({ length: 10 })}`;
+        //Set filter by key type and filter per key name
+        await browserPage.searchByKeyName(keyName);
+        await browserPage.selectFilterGroupType(COMMAND_GROUP_SET);
+        //Verify the clear control
+        await t.click(browserPage.clearFilterButton);
+        await t.expect(browserPage.multiSearchArea.find(browserPage.cssFilteringLabel).visible).notOk('The filter per key type is removed');
+        await t.expect(browserPage.filterByPatterSearchInput.getAttribute('value')).eql('', 'All characters from filter input are removed');
+        await t.expect(browserPage.clearFilterButton.visible).notOk('The clear control is disappeared');
+    });
+test
+    .after(async() => {
+        //Delete database
+        await deleteDatabase(ossStandaloneConfig.databaseName);
+    })
+    .meta({ rte: rte.standalone })
+    ('Verify that when user clicks on “clear” control and filter per key name is applied filter is reseted and rescan initiated', async t => {
+        keyName = `KeyForSearch${chance.word({ length: 10 })}`;
+        //Set filter by key name
+        await browserPage.searchByKeyName(keyName);
+        //Verify the clear control
+        await t.click(browserPage.clearFilterButton);
+        await t.expect(browserPage.keyListTable.textContent).contains('No keys to display.', 'The rescan initiated');
+        await t.expect(browserPage.filterByPatterSearchInput.getAttribute('value')).eql('', 'The filtering is reseted');
+    });
+test
+    .after(async() => {
+        //Delete database
+        await deleteDatabase(ossStandaloneConfig.databaseName);
```

Comment:
You already have it in the fixture hook level

Reply:
yes but in the fixture hook the key removing step exists, thats not needed in this test